content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -49,29 +45,34 @@ this:
 [compare]: /doc/user-guide/experiment-management/comparing-experiments
 [persistent]: /doc/user-guide/experiment-management/persisting-experiments
 
-## Initialize DVC Experiments on any project
+## Save experiments
 
-To use DVC Experiments you need a <abbr>DVC project</abbr> with a minimal
-structure and configuration. To avoid having to bootstrap DVC manually, the
-`dvc exp init` command lets you quickly onboard an existing project to the DVC
-Experiments workflow.
+DVC is not opinionated about your experiments workflow. To save an experiment,
+you may choose any of these workflows:
 
-It will create a simple `dvc.yaml` metafile, which codifies your planned
-experiments. This includes the locations for expected <abbr>dependencies</abbr>
-(data, parameters, source code) and <abbr>outputs</abbr> (ML models,
-<abbr>metrics</abbr>, etc.). These assume [sane defaults] but can be customized
-with the options of `dvc exp init`.
+- If you do not have a DVC pipeline, you can add [DVCLive] to your Python code
+  to log live results and save the experiment by including the keyword argument
+  `save_dvc_exp=True`.
```

Comment:
💅🏼 
```suggestion
- If you do not have a DVC pipeline, you can log live results from Python code
  [using DVCLive], initialized with `save_dvc_exp=True`.
```
⚠️ MD link ref would need the other half of the update (`[using dvclive]`).